doc/man7/provider-asym_cipher.pod

```diff
@@ -234,14 +234,14 @@ The TLS protocol version first requested by the client.
 
 The negotiated TLS protocol version.
 
-=item "implicit-rejection" (B<OSSL_PKEY_PARAM_IMPLICIT_REJECTION>) <unsigned integer>
+=item "implicit-rejection" (B<OSSL_ASYM_CIPHER_PARAM_IMPLICIT_REJECTION>) <unsigned integer>
 
 Gets of sets the use of the implicit rejection mechanism for RSA PKCS#1 v1.5
```

Comment:
Please fix the of -> or typo here when you're touching this.

Reply:
Done